docs/src/components/ReactExample/Example.tsx

```diff
@@ -143,7 +143,7 @@ const Example = ({
   }, [background, setSelectedBackground, handleKeyDown, exampleName]);
 
   const content = (
-    <StyledWrapper isFullScreen={isFullScreen}>
+    <StyledWrapper isFullScreen={isFullScreen} dir="ltr">
```

Comment:
This is an attempt to fix the broken react example https://orbit.kiwi/components/progress-indicators/wizard/